src/gpio/partially_erased.rs

```diff
@@ -0,0 +1,129 @@
+use super::*;
+
+pub type PEPin<const P: char, MODE> = PartiallyErasedPin<P, MODE>;
```

Comment:
As the other type alias, I'm not a fan of this one. Though here the case is stronger for an alias as the original type name is long. 

But I don't think this is a problem. A user might just use a type alias themself via `use hal::gpio::PartiallyErasedPin as PEPin`.